packages/govuk-frontend/rollup.release.config.mjs

```diff
@@ -30,7 +30,13 @@ export default defineConfig(({ i: input }) => ({
     plugins: [
       terser({
         format: { comments: false },
-        mangle: { reserved: Object.keys(GOVUKFrontend) },
+        mangle: {
+          keep_classnames: true,
+          keep_fnames: true,
+          // Keep our exports reserved, which will make sure any `const`,
+          // like `version` remains untouched
```

Comment:
Worth clarifying that it's not just _any_ `const` that remains untouched

It's limited to all top-level exports, in whatever form they take

```suggestion
          // Ensure all top-level exports skip mangling, for example
          // non-function string constants like `export { version }`
```